docs(consumer_group): add delete api for consumer group configuration (#4785)
Kong/docs.konghq.com: src/gateway/kong-enterprise/consumer-groups/index.md

    }
    ```

## Delete consumer group configurations

> If you still need the consumer group but wanted to delete the configuations, 
> you can delete it. This removes configurations for the consumer group.

Reviewer: ```suggestion
You can also clear the configuration of a consumer group without deleting the consumer group itself.
```

> The consumers in the group are not deleted and are still in the consumer group.

Reviewer: ```suggestion

With this method, the consumers in the group aren't deleted and are still in the consumer group.
```

1. Delete the consumer group configuration using the following request:

{% capture delete_consumer_group_config %}
{% navtabs codeblock %}
{% navtab cURL %}
```bash

> curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \

Reviewer: ```suggestion
curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced
```

```
{% endnavtab %}
{% navtab HTTPie %}
```bash

> http DELETE :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \

Reviewer: ```suggestion
http DELETE :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced
```

```
{% endnavtab %}
{% endnavtabs %}
{% endcapture %}

> {{ delete_consumer_group_config| indent | replace: " </code>", "</code>" }}

Reviewer: ```suggestion
{{ delete_consumer_group_config | indent | replace: " </code>", "</code>" }}
```

    If successful, you receive see the following response:
    ```
    HTTP/1.1 204 No Content
    ```
    
1. To verify, check the consumer object configuration:

{{ check_group1 | indent | replace: " </code>", "</code>" }}

>     Response, notice the plugins object in the response is gone.

Reviewer: ```suggestion
    Response, without a `plugins` object:
```
